content/rosa/terraform/classic.md

```diff
@@ -0,0 +1,79 @@
+---
+date: '2024-05-20'
+title: Deploying a ROSA Classic cluster with Terraform
+tags: ["ROSA", "terraform"]
+authors:
+  - Paul Czarkowski
+---
+
+This guide will walk you through deploying a ROSA cluster using Terraform.  This is a great way to get started with ROSA and to automate the deployment of your clusters.
+
+{{< readfile file="/content/rosa/terraform/tf-prereqs.md" markdown="true" >}}
+
+## HCP ROSA Cluster
+
+1. Clone down the terraform repository
+
+    ```bash
+    git clone --depth=1 https://github.com/rh-mobb/terraform-rosa.git
+    cd terraform-rosa
+    ```
+
+1. Save some environment variables
+
+    > Note: You may want to customize some of these settings to match your needs. see the `variables.tf` file for options.
+
+    ```bash
+    export TF_VAR_token="$(jq -r .refresh_token ~/.config/ocm/ocm.json)"
+    export TF_VAR_cluster_name="$(whoami)"
+    export TF_VAR_admin_password='Passw0rd12345!'
+    export TF_VAR_developer_password=''
+    export TF_VAR_private=false
+    export TF_VAR_ocp_version=4.15.11
+    export TF_VAR_hosted_control_plane=false
```

Comment:
Why have two nearly identical guides for HCP and Classic that mostly differ based upon whether this parameter is true or false?